installation/sql/postgresql/base.sql

```diff
@@ -1229,6 +1229,7 @@ CREATE TABLE IF NOT EXISTS "#__workflow_stages" (
   "title" varchar(255) NOT NULL,
   "description" text NOT NULL,
   "default" smallint NOT NULL  DEFAULT 0,
+  "position" text DEFAULT NULL,
```

Comment:
```suggestion
  "position" text,
```
DEFAULT NULL is not needed as that is the default.